R/add_phylopic.r

```diff
@@ -70,7 +77,7 @@
 #' p + ggtitle("R Cat Herd!!")
 add_phylopic <- function(img = NULL, name = NULL, uuid = NULL,
                          x, y, ysize = Inf,
-                         alpha = 1, color = "black",
+                         alpha = 1, color = "transparent", fill = "black",
```

Comment:
The current default for `fill` here and in `add_phylopic` is "black". Perhaps this should be NULL or not have a default as user's old code will now not produce the same results as before, i.e. `color` will now just give them a coloured outline rather than recolouring the whole image. In this way, we could use `color` and pass to `fill` if `fill` is NULL.